testfixtures/__init__.py

```diff
@@ -1,16 +1,19 @@
 # Copyright (c) 2008-2014 Simplistix Ltd
 # See license.txt for license details.
 
+
 class singleton(object):
     def __init__(self, name):
         self.name = name
+
     def __repr__(self):
         return '<%s>' % self.name
     __str__ = __repr__
-    
+
 not_there = singleton('not_there')
 
-from testfixtures.comparison import Comparison, StringComparison, RoundComparison, compare, diff
+from testfixtures.comparison import (Comparison, StringComparison,
+                                     RoundComparison, compare, diff)
 from testfixtures.tdatetime import test_datetime, test_date, test_time
```

Comment:
Sure, can change this. My reasoning here is the same as for imports (even though for imports it's actually a pep8 error): your diffs are easier to read through time.